assert-env.js

```
@@ -0,0 +1,10 @@
+const assertHerokuEnv = require('@quarterto/assert-heroku-env');
+const logger = require('./logger');
+const sentry = require('./sentry').init();
+
+if (process.env.NODE_ENV === 'production') {
+	assertHerokuEnv(warnings => {
+		logger.error({ event: 'MISSING_ENV_VARS', warnings });
+		sentry.captureException(warnings, { level: 'warning' });
```

Should we raise this as an `error` level instead of `warning`?